skiko/src/jvmMain/kotlin/org/jetbrains/skia/impl/RefCnt.jvm.kt

```diff
@@ -23,9 +23,9 @@ actual abstract class RefCnt : Managed {
         val PTR = _nGetFinalizer()
     }
 
-    companion object {
+    actual companion object {
         @JvmStatic
-        external fun _nGetFinalizer(): NativePointer
+        actual external fun _nGetFinalizer(): NativePointer
```

Comment:
Better move those on top level for consistency.

Reply:
for some reason in the JVM RefCnt it was already in a companion object so I did this for consistency - otherwise we change slightly signature on JVM - insignificantly though but sure, I can do it.

Reply:
pushed all requested changes